sdk/react/src/lib/fragment-components/a-time-picker.tsx

```diff
@@ -0,0 +1,360 @@
+	</>;
+};
+
+export const ATimePickerCodeUI = ({ selectedComponent, setComponent }: any) => <TextInput
+	value={selectedComponent.codeContext?.name}
+	labelText='Input name'
+	onChange={(event: any) => setComponent({
+		...selectedComponent,
+		codeContext: {
+			...selectedComponent.codeContext,
+			name: event.currentTarget.value
+		}
+	})}
+/>;
+
+export const ATimePicker = ({
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		rejectDrop={true}
+		{...rest}>
+			{
+				<TimePicker
+				disabled={componentObj.disabled}
+				invalid={componentObj.invalid}
+				invalidText={componentObj.invalidText}
+				placeholder={componentObj.placeholder}
+				hideLabel={componentObj.hideLabel}
+				labelText={componentObj.label}
+				light={componentObj.light}
+				size={componentObj.size}>
+					<TimePickerSelect disabled={componentObj.disabled} labelText="time-picker-1">
+						<SelectItem value="AM" text="AM" />
+						<SelectItem value="PM" text="PM" />
+					</TimePickerSelect>
+					<TimePickerSelect disabled={componentObj.disabled} labelText="time-picker-2">
+						{
+							componentObj.items.map((step: any, index: number) => <SelectItem
+								value={step.value}
+								text={step.text}
+								key={index}
+							/>)
+						}
+					</TimePickerSelect>
+				</TimePicker>
+			}
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: ATimePicker,
+	settingsUI: ATimePickerSettingsUI,
+	codeUI: ATimePickerCodeUI,
+	keywords: ['timepicker', 'time', 'picker'],
+	name: 'Time Picker',
+	type: 'time-picker',
+	defaultComponentObj: {
+		type: 'time-picker',
+		disabled: false,
+		invalid: false,
+		invalidText: 'A valid value is required',
+		placeholder: 'hh:mm',
+		label: 'Select a time',
+		hideLabel: false,
+		light: false,
+		size: 'md',
+		value: '',
+		items: [
+			{
+				value: 'timezoneValue1',
+				text: 'Time zone 1'
+			},
+			{
+				value: 'timezoneValue2',
+				text: 'Time zone 2'
+			}
+		]
+	},
+	image,
+	codeExport: {
+		angular: {
+			latest: {
+                inputs: ({ json }) => `@Input() ${nameStringToVariableString(json.codeContext?.name)}Value = "${json.value}";`,
+                outputs: ({ json }) => `@Output() ${nameStringToVariableString(json.codeContext?.name)}ValueChange = new EventEmitter();`,
+                imports: ['TimePickerModule', 'TimePickerSelectModule'],
+                code: ({ json }) => {
+                    return `<ibm-timepicker
+                        ${angularClassNamesFromComponentObj(json)}
+                        label="${json.label}"
+                        ${json.light ? '[theme]="light"' : '[theme]="dark"'}
+                        [invalid]="${json.invalid}"
+                        [placeholder]="'${json.placeholder}'"
+                        [size]="'${json.size}'"
+                        [hideLabel]="${json.hideLabel}"
+                        [invalidText]="'${json.invalidText}'"
```

Comment:
Once these are set, users will not be able to update them. These need inputs. Also, you're binding them to variables, not values.